api/filter_outputs.go

```diff
@@ -9,6 +10,42 @@ import (
 	"github.com/pkg/errors"
 )
 
+func (api *API) getFilterOutput(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	fID := chi.URLParam(r, "filter-output-id")
+
+	var filterOutput *model.FilterOutput
+
+	filterOutput, err := api.store.GetFilterOutput(ctx, fID)
+	if err != nil {
+		api.respond.Error(
+			ctx,
+			w,
+			statusCode(err),
+			Error{
+				err: errors.Wrap(err, "failed to get filter output"),
+				logData: log.Data{
+					"id": fID,
+				},
+			},
+		)
+		return
+	}
+
+	resp := getFilterOutputResponse{
+		filterOutput.FilterID,
```

Comment:
If I'm not mistaken, the getFilterOutput response equates to simply a FilterOutput resource. The simplest way to do this is to define the GetFilterOutputResponse as simply haiving the FilterOutput embedded into it, and you can assign it as `resp := getFilterOutputResponse{FilterOutput: filterOutput}`. This is done a few times in the other endpoints for an example

Reply:
In light of the new swagger sped, this now makes sense, I have done so in the last commit, and amended feature tests to reflect the new response. I will mark as resolved.